Sources_v3/StreamChat/Models/Message.swift

```diff
@@ -109,6 +109,8 @@ public struct _ChatMessage<ExtraData: ExtraDataTypes> {
     
     /// The entire list of reactions to the message left by the current user.
     public let currentUserReactions: Set<_ChatMessageReaction<ExtraData>>
+    
+    public let isSentByCurrentUser: Bool
```

Comment:
This looks iffy. Let's see if we can get rid of it when working on this in detial.